portal-ui/src/screens/Console/Common/TabSelector/TabSelector.tsx

```diff
@@ -27,27 +27,39 @@ interface ITabSelector {
 }
 
 const tabSubStyles = makeStyles({
+  tabRoot: {
+    height: "40px",
+    borderBottom: "1px solid #eaeaea",
+  },
   root: {
+    width: "120px",
     backgroundColor: "transparent",
     paddingTop: 0,
     paddingBottom: 0,
-    fontSize: 22,
-    textTransform: "uppercase",
-    color: "#D0D0D0",
+    fontSize: "14px",
+    fontWeight: 600,
+    color: "#07193E",
+    height: "40px",
+  },
+  selected: {
+    backgroundColor: "#F6F7F7 !important",
```

Comment:
Please not use `!important`

I think this component has some class that can be overriden Eg.

```suggestion
   "&.MuiTab-selected": {
     backgroundColor: "#F6F7F7 !important",
   }
    
```